os/logm/logm_process.c

```diff
@@ -42,10 +42,12 @@ static int logm_change_bufsize(int buflen)
 	}
 
 	/* Realloc new buffer with new length */
-	g_logm_rsvbuf = (char *)realloc(g_logm_rsvbuf, buflen);
-	if (!g_logm_rsvbuf) {
+	char *new_g_logm_rsvbuf = (char *)realloc(g_logm_rsvbuf, buflen);
+	if (new_g_logm_rsvbuf == NULL) {
+		dbg("Realloc Fail\n");
```

Comment:
It is duplicated because logm task show fail message when this failed.
please see line93 in logm_process.c

Reply:
For the log msg I put under dbg to specify only realloc failure
Line 93 is common for any error ....

Reply:
But this fail is a only failed case in this funtion.